code/js/rcalc-keypress.js

```diff
@@ -0,0 +1,107 @@
+var isSecondPressed = false;
+var collectionOfKeys = [];
+var nextToLastCharacter;
+
+document.addEventListener("keyup", function(event) {
+  if (event.defaultPrevented) {
+    return;
+  }
+  var key = event.key;
+  collectionOfKeys.push(key);
+  nextToLastCharacter = collectionOfKeys[collectionOfKeys.length - 2];
+  switch (key) {
+    case "0":
```

Comment:
Please create constant file for all your checked keys. 
rcalc-keys.OPERATOR_PLUS
rcalc-keys.NUMBER_1
rcalc-keys.KEY_M
...

Reply:
created file string-key-keypress-constant.js ... all keys used in this functionality are constants now